### PR TITLE
README.md: improved wording

### PR DESCRIPTION
**Pull request by mariomadproductions**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -338,4 +338,4 @@ Account 6:
 - Monopoly
 - Peggle
 
-You can sync games only from 5 different owners at once (Apple mad DRM limitation unfortunately). Consider each account ID as a games pack and consider that you can get only 5 different games packs into your device. This limitation may be rendered less annoying if I can preserve a lot of games on one distinct account. You need to choose your preferred games and avoid games packs that contains games that you will not really play.
\ No newline at end of file
+You can sync games only from 5 different owners at once (Apple mad DRM limitation unfortunately). Consider each account ID as a games pack and consider that you can get only 5 different games packs into your device. This limitation may be rendered less annoying if I can preserve a lot of games on one distinct account. You need to choose your preferred games and avoid games packs that contains games that you will not really play.
```
Comment (Olsro) [inline README.md:341]: No changes here ?

**Comment by BuzzerBeaterClutch** [comment]
image

Wanted to comment on the command

``7za x "iPod Clickwheel Games Preservation Project.utm.7z.001"``

should have  dots in place of the spaces like so

``7za x "iPod.Clickwheel.Games.Preservation.Project.utm.7z.001"``

This is on Ubuntu 24.04

File: README.md (hunk 1 above)
Comment (mariomadproductions) [reply README.md:341]: > No changes here ?

Yeah, not sure why its showing up in the diff.

**Comment by mariomadproductions** [comment]
> I've done the changes myself, thanks for the proposals

Thanks. Although "SHA signatures" doesn't really make sense. I think it should probably say "SHA-256 hashes".

**Comment by Olsro** [comment]
Done

**Pull request closed without merging**